DI-Policies.md

```diff
@@ -0,0 +1,51 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+
+   + Speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website.
+
+   + Monthly for social.
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
+
++ Continuing education 
+
+   + Minimum two PAID workshops a year (pay Black folx and other marginalized individuals for their work).
+
+   + Before participation in any styled shoots, workshops, or other events, inquire into their own D+I policies. 
```

Comment:
Should you write what action you will take if you feel the D+I policies are not up to par with your own standards?

Reply:
I'm not sure this falls into `Continuing Education`? Is the goal to learn from others in the industry, or to ensure that their standards are aligned with your own? If it's the latter, I might move this into `Vetting subcontractors + partners`

Reply:
@Pseudopam Yes, good point. I will expand here to make it clear.

@bryanaknight styled shoots and workshops are usually where/how I get most of my education around creative processes, marketing, networking, etc for the industry. So, I put this item here because of that, though the commitment is specifically related to ensuring I'm only attending (and giving my money to) styled shoots and workshops that have good D+I policies and align with my values. I'm not sure I'd want it under `Vetting subcontractors...` because these aren't subcontractors or partners - they are rather folks with their own businesses/services that I'm purchasing. Knowing that, I still feel like it goes here, but what do you think?

Reply:
That makes sense! Thanks for clarifying @nadiajoyce